### PR TITLE
Bioinfo Tab: Show edit history and load page faster

### PR DESCRIPTION
**Pull request by kate-v-stepanova**
Deployed on stage: https://genomics-status-stage.scilifelab.se/bioinfo/P4725

Before merging, needs this one to be merged first: https://github.com/SciLifeLab/StatusDB_views/pull/58

### CODE REVIEW

**Comment by ewels** [comment]
Haven't looked at the code yet, but I found a bug - if you expand a row and minimise it again, all rows are hidden..

**Comment by ewels** [comment]
Also it would be good to set a `min-width` on the date columns so that the fields don't overflow:

image

**Comment by ewels** [comment]
Also, on stage everything is disabled (for me, anyway). However, the _set all_ buttons for date (X and today) do change the fields.

**Comment by kate-v-stepanova** [comment]
was not able to reproduce the bug. Can you tell me the project number and which row did you try to expand?
This also can happen, if the page was not completely loaded, and when the event is triggered, there is js still doing something else, so it's not processed properly.

**Comment by kate-v-stepanova** [comment]
yes, this is known bug, it's on to-fix list

**Comment by ewels** [comment]
The link you posted above. Not related to incomplete loading, I can wait ages and it still does it.

ezgif com-video-to-gif 1

**Comment by kate-v-stepanova** [comment]
Interesting. Never saw this before. Looking into it.

**Comment by ewels** [comment]
Let me know when this is redeployed on stage / ready for review..

**Comment by kate-v-stepanova** [comment]
it's redeployed, but I'm still looking into delivery date thing

**Comment by kate-v-stepanova** [comment]
a good example to check if it works as intended: https://genomics-status-stage.scilifelab.se/bioinfo/P4604

File: run_dir/design/bioinfo_tab.html [hunk 1]
```diff
@@ -42,9 +42,7 @@
     {% include "bioinfo_tab/run_lane_sample_view.html" %}
     {% include "bioinfo_tab/sample_run_lane_view.html" %}
 
-    <button disabled class="btn btn-default" id="bioinfo-show-history">Show Edit History</button>
-    <pre id="bioinfo-history-dump" style="display:none; margin-top:20px;"></pre>
-
+    <button id="bioinfo-show-history-button" class="btn btn-default" data-target="#bioinfo-show-history">Show Edit History</button>
     <script src="/static/js/bootstrap-datepicker.min.js"></script>
```
Comment (ewels) [inline run_dir/design/bioinfo_tab.html]: Would be good to move these to the end of the file, might get lost in the middle here.

Comment (kate-v-stepanova) [inline run_dir/design/bioinfo_tab.html]: done!

**Pull request merged**